packages/fxa-settings/src/pages/ResetPassword/index.tsx

```diff
@@ -42,19 +39,21 @@ type FormData = {
 
 // eslint-disable-next-line no-empty-pattern
 const ResetPassword = ({
-  serviceName,
+  // TODO: Pull service name from Relier model FXA-6437
+  serviceName = MozServices.Default,
   prefillEmail,
   forceAuth,
-  canGoBack,
 }: ResetPasswordProps & RouteComponentProps) => {
-  usePageViewEvent(viewName, REACT_ENTRYPOINT);
+  logPageViewEvent(viewName, REACT_ENTRYPOINT);
```

Comment:
I fix the issue with this [here](https://github.com/mozilla/fxa/pull/14844#discussion_r1098056157) but not sure it's the best long term solution. You could change this to `use` when you pull service name from the relier.